reduce sending of state when not nessesary

Xento opened the pull request:
The state message only needs to be send when there is no motion send
during the interval.
The state interval can be reset when a motion message was send.

Additionaly 5 minutes for the interval seems more than the original
homematic PIR (compared to a HM-Sen-MDIR-O-2)

pa-pa commented:
Change the alarm tick when it is active is not allowed, because all alarms are managed relative to each other. So add some ticks to an alarm will also add the same amount to all other alarms after the changed. Alarm needs to cancel before changed. So we can simply add the following code to the end of the trigger method.

    // cancel cyclic message and restart
    sysclock.cancel(cycle);
    cycle.set(LIGHTCYCLE);
    sysclock.add(cycle);

Not tested.

Xento commented:
I fixed it.
You can't move the reset to the end of the trigger because when you set it at the end a motion just before the end of the quiet time would reset the alarm.

1. Motion detected send, quiet for 4 minutes, cyclic message in 5 minutes
2. Motion detected, still quiet for 1 minute, cylic message reset to 5 minutes
3. No motion detected by timeout on ccu, cyclic message in 4 minutes
5. cyclic message send after 9 minutes

I removed the reset at the else if, because the higher cycle time
